keyboard/app.js

```diff
@@ -0,0 +1,14 @@
+import {setJiggleToAnotherKey,
+    keyCheck,
+    preventException} from './utility.js';
+let listOfAllKeysByClassName = document.getElementsByClassName("key");
+
+window.addEventListener("keydown", (event)=>{
+    
+    let keyVal = event.key.toString().toUpperCase();
+    preventException(event, keyVal); 
+    keyCheck(keyVal);
+});
+export {
+    listOfAllKeysByClassName
```

Comment:
you should not export any function from your main js file

Reply:
This is not a function but a classList variable, is it fine to export variables